exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/ParquetRecordReader.java

```diff
@@ -105,13 +107,15 @@
   long totalRecordsRead;
   private final FragmentContext fragmentContext;
 
+  public ParquetReaderStats parquetReaderStats = new ParquetReaderStats();
```

Comment:
should we mark this as `final` ? it makes it easier for other developers understand how it's supposed to be used. Of course, we'll have to remove the `set to null` in the `close()` method but I guess that's fine unless of course the close method can be called multiple times